config/routes.rb

```diff
@@ -12,6 +12,7 @@
   resources :orders, only: [:index, :show]
   resources :cart_items, only: [:create, :delete, :show]
   get '/cart_items', to: "cart_items#show", as: "cart"
+  patch '/cart_items', to: "cart_items#increment", as: "increment_cart_items"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.